docs/reference/rollup/apis/delete-job.asciidoc

```diff
@@ -11,6 +29,8 @@ Deletes an existing {rollup-job}.
 
 experimental[]
 
+endif::[]
```

Comment:
I'm a little confused. is the above header also in a second `ifndef::permanently-unreleased-branch`?

Reply:
Sorry. I should have explained this better.

There are two conditionals: `ifdef` and `ifndef`.  The content inside `ifdef` only displays on the master and 7.x branches (permanently unreleased branches). The content inside `ifndef` displays on all the other branches. Content outside of those conditionals displays on any branch containing the source.

This lets us use the same source when 7.n is cut from 7.x. Though the source will be the same, 7.x will use the `ifdef` content while 7.n will use the `ifndef` content.

When we get near release, I'll go through the docs and remove all the conditionals so only the `ifdef` content is left. I have a task for that over in #65515.

Reply:
ahh ok. thanks for the clarification